src/common/mmap.c

```diff
@@ -397,8 +397,11 @@ util_range_split(struct map_tracker *mt, const void *addrp, const void *endp)
 	uintptr_t addr = (uintptr_t)addrp;
 	uintptr_t end = (uintptr_t)endp;
 	ASSERTne(mt, NULL);
-	ASSERTeq(addr % Mmap_align, 0);
-	ASSERTeq(end % Mmap_align, 0);
+	if (addr == end || addr % Mmap_align != 0 || addr % Mmap_align != 0) {
```

Comment:
"**addr % Mmap_align != 0**" two times in if condition ?

Reply:
;)